plugins/baseball-scoreboard/sports.py

```
@@ -265,13 +265,30 @@ def _load_custom_font_from_element_config(self, element_config: Dict[str, Any],
+                    # FreeType only accepts the exact pixel size baked into the
+                    # file (its "strike") and raises "invalid pixel size" for
+                    # anything else. Try the requested size first (in case it
+                    # happens to match), then fall back to the file's real
+                    # native size read from its SIZE header line, so a BDF
+                    # font can still be selected via font_size-driven configs
+                    # without the caller needing to know its exact strike size.
                     try:
                         font = ImageFont.truetype(font_path, font_size)
                         self.logger.debug(f"Loaded BDF font: {font_name} at size {font_size}")
                         return font
                     except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)